spec/rails_helper.rb

```diff
@@ -89,4 +88,17 @@
   Capybara.add_selector(:data_output) do
     xpath { |id| XPath.css("[data-output='#{id}']") }
   end
+
+  # copied from Sequencescape but removed the extra bits not needed currently.
+  Capybara.register_driver :headless_chrome do |app|
```

Comment:
Without the extra bits, these aren't needed and are part of capybara itself, albeit with the less convenient names of selenium_chrome and selenium_headless.

https://github.com/teamcapybara/capybara/blob/master/lib/capybara/registrations/drivers.rb